smart_str: mention smart_str_extract() (#127)
phpinternalsbook/PHP-Internals-Book: Book/php7/internal_types/strings/smart_str.rst

    /* Don't forget to release/free it */
    smart_str_free(&my_str);

> We can also use the embedded ``zend_string`` independently of the ``smart_str``::

Reviewer: ```suggestion
It is also possible to embedded the ``zend_string`` independently of the ``smart_str``::
```
Although I don't know if there is a specific style guide about not using "we"

Reviewer 2: There are already 4 occurrences of "we" on that page, so I guess it is good as is.

    smart_str my_str = {0};

    smart_str_appends(&my_str, "Hello, you are using PHP version ");
    smart_str_appends(&my_str, PHP_VERSION);

    zend_string *str = smart_str_extract(my_str);

>     RETURN_STRING(str);

Reviewer: ```suggestion
    RETURN_STR(str);
```
``RETURN_STRING()`` is for `char*`

Author: Thank you

>     /* We don't need to free my_str in this case */

Reviewer 2: Maybe clarify that we *must* not free:
```suggestion
    /* We must not free my_str in this case */
```

Author: Thank you
